frontend/src/features/crawl-workflows/workflow-editor.ts

```diff
@@ -1970,6 +1971,7 @@ https://archiveweb.page/images/${"logo.svg"}`}
     }
 
     const config = this.parseConfig();
+    console.log(config);
```

Comment:
console log snuck in!

Reply:
Oh yeah the whole frontend commit is to throw out!

Reply:
Does it make sense to keep the frontend commit, but keep the default `selectLinks` to `a[href]->href`?

Reply:
Yeah not a bad idea, just did that and added `selectLinks` to the workflow config typing more properly. Should give you a better base later on to build on.